src/components/shoppingListCreateForm/shoppingListCreateForm.module.css

```diff
@@ -0,0 +1,91 @@
+.root {
+  text-align: center;
+  margin: 0 auto 16px;
+  width: 100%;
+}
+
+.fieldset {
+  border: none;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+}
+
+.input {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  padding: 8px 12px;
+  margin-right: 12px;
+  background-color: #fff;
+  color: #000;
+  border: 1px solid #5f5f5f;
+  border-radius: 3px;
+}
+
+.input:disabled {
+  opacity: 0.7;
+  cursor: not-allowed;
+}
+
+.button {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  color: var(--button-text-color);
+  background-color: var(--button-color);
+  border: 1px solid var(--button-border-color);
+  border-radius: 3px;
+  padding: 8px 12px;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: var(--button-hover-color);
+}
+
+.button:disabled,
+.button:disabled:hover {
+  background-color: #7c8388;
+  border-color: #4b5053;
+  cursor: not-allowed;
+}
+
+@media (min-width: 481px) {
+  .root {
+    --button-width: 72px;
+    margin: unset;
+    margin-bottom: 16px;
+    width: 228px;
+  }
+
+  /**
+   *
+   * 228px: width of games dropdown - form should be the same width
+   *        on small tablet
+   * 24px:  side padding on input
+   * 12px:  right margin on input
+   *
+   */
+  .input {
+    width: calc(228px - 24px - 12px - var(--button-width));
+  }
+
+  .button {
+    width: var(--button-width);
+  }
+}
+
+@media (min-width: 601px) {
+  .root {
+    margin: 0 auto 24px;
+    width: unset;
```

Comment:
Instead of all these `unset`s, I tried making the above media query 

```css
@media (min-width: 481px) and (max-width: 600px)
```

But it turned out that it also needed to be unset from the value for < 481px. This could be risky since it will mean these values depend on the user-agent stylesheet so I might try and find out the values in Brave and set them explicitly to those.